gbfs.md

```diff
@@ -816,13 +812,14 @@ Field Name | REQUIRED | Type | Defines
         "capacity": 16,
         "vehicle_type_area_capacity": [
```

Comment:
Change the field name to `vehicle_types_capacity` for consistency with `vehicle_types_available` in [station_status.json](https://github.com/MobilityData/gbfs/blob/master/gbfs.md#station_statusjson) and with `vehicle_docks_capacity` above.

```suggestion
        "vehicle_types_capacity": [
```